app/models/concerns/convertable.rb

```diff
@@ -0,0 +1,12 @@
+module Convertable
+  extend ActiveSupport::Concern
+
+  def convert_date_id(attributes_hash)
+    attributes_hash.tap do |hash|
+      attributes_hash.each_key do |attr_key|
+        hash[attr_key] = send(attr_key.to_sym).iso8601 if attribute(attr_key).kind_of?(Time) && send(attr_key.to_sym)
```

Comment:
`#send` can take a string. You don't need to convert to symbol.

Reply:
we could also use `respond_to?(attr_key)` instead of looking for nil coming back from `send`

Reply:
@lindgrenj6 @bzwei Changed.